backend/packages/Upgrade/.env.example

```diff
@@ -22,6 +22,7 @@ LOG_OUTPUT=dev
 #
 TYPEORM_CONNECTION=postgres
 TYPEORM_HOST=localhost
+TYPEORM_HOSTNAME_EXPORT_REPLICA=[]
```

Comment:
is the separate new env var needed for the typeorm update? does it need to be an array if it's just one thing?

Reply:
No, it was part of improvement we found while updating replica connection syntax during this PR. Its kept as an array for consistency with the other replica hostnames var. We can make it a string if you want.

Reply:
ok, this complicates promoting this to production as it's an unexpected configuration change that I don't remember being talked about. @bcb37 @VivekFitkariwala what do you think